.changeset/bright-dodos-punch.md

```diff
@@ -0,0 +1,7 @@
+---
+'@roadiehq/catalog-backend-module-aws': major
```

Comment:
Major should be reserved for breaking changes. If there is a breaking change we should list it here. If not this should be reduced to minor or patch.